src/olympia/addons/templates/addons/impala/persona_grid.html

```diff
@@ -10,6 +10,12 @@
               <img {{ 'src' if first_page else 'data-defer-src' }}="{{ addon.persona.thumb_url }}"
                    data-browsertheme="{{ addon.persona.json_data }}" alt="">
             </div>
+            <div class="persona-install">
```

Comment:
I can’t tell if it was present on the homepage looking back, but it should have been there (it’s useful to have).

It’s hard to trace because this file has only existed since 580de6f78ea89b10ae85763fe169e873491de58f